frontend/package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "xchemlab-frontend",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+      "@testing-library/jest-dom": "^5.17.0",
+      "@testing-library/react": "^13.4.0",
+      "@testing-library/user-event": "^13.5.0",
+      "react": "^18.2.0",
+      "react-dom": "^18.2.0",
+      "web-vitals": "^2.1.4"
+    },
+    "devDependencies": {
+      "eslint": "8.49.0",
+      "eslint-config-react-app": "^7.0.1",
+      "eslint-webpack-plugin": "^4.0.1",
+      "prettier": "3.0.3",
+      "react-scripts": "5.0.1",
+      "typescript": "5.2.2"
+    },
+    "scripts": {
+      "start": "react-scripts start",
+      "build": "react-scripts build",
+      "test": "react-scripts test",
+      "eject": "react-scripts eject",
+      "format": "prettier --jsx-single-quote --print-width 100 --write src/**/*.{ts,tsx,css}"
```

Comment:
Would rather we stuck with the defaults, or defined any changes in a `.prettierrc` file if we want to deviate